app/sidekiq/lighthouse/pension_benefit_intake_job.rb

```diff
@@ -33,19 +38,15 @@ class PensionBenefitIntakeError < StandardError; end
     #
     # @param [Integer] saved_claim_id
     # rubocop:disable Metrics/MethodLength
-    def perform(saved_claim_id)
+    def perform(saved_claim_id, user_uuid = nil)
```

Comment:
Now that we're using a Sidekiq job that is specific to Pensions, we can update to pass the user_uuid

Reply:
Any benefit to passing the entire user object, not just the uuid?

Reply:
See comment above (passing just the ID is recommended for the Sidekiq arguments)